include/openssl/evp.h

```diff
@@ -427,6 +427,19 @@ struct evp_cipher_st {
 # define         EVP_CTRL_TLS1_1_MULTIBLOCK_DECRYPT      0x1b
 # define         EVP_CTRL_TLS1_1_MULTIBLOCK_MAX_BUFSIZE  0x1c
 
+/* EVP_CTRL_SET_SBOX takes the char* specifying S-boxes */
+# define         EVP_CTRL_SET_SBOX                       0x1d
+/* 
+ * EVP_CTRL_SBOX_USED takes a 'size_t' and 'char *', pointing at a
+ * pre-allocated buffer with specified size 
+ */
+# define         EVP_CTRL_SBOX_USED                      0x1e
+/* EVP_CTRL_KEY_MESH takes 'size_t' number of bytes to mesh the key after, 0 switches meshing off */
+# define         EVP_CTRL_KEY_MESH                       0x1f
+/* EVP_CTRL_BLOCK_PADDING_MODE takes the char* specifying padding mode */
```

Comment:
Speaking from a general point of view, I think specifying them as strings is a good thing. It avoids the hassle of having to define symbols or the use of anonymous numbers, and truly becomes a communication between the app and the underlying algo with none else (OpenSSL itself included)  being the wiser.